### ISSUE TITLE
Site logo looks off when editing site (templates, etc.) only

### ISSUE THREAD

**Issue by janboddez** [WordPress/gutenberg#53148]
### Description

The icon fills up all the space (while the "focus box" does not, see screenshot). On other screens, like the Edit Post one, it still looks like before, i.e., smaller. (Again, see screenshot.)

Actually, it looks OK on the "template selection" screen as well; it's the actual Site Editor screens (where you edit the templates, etc.) that are "different."

In fact, some of the icon is cut off along all four sides (it is hard to see).

I understand this may have been done on purpose and can actually look okay for certain images (photos with some of margin around the "main subject") but it came as quite a surprise.

### Step-by-step reproduction instructions

I had a site logo set (using the Site Logo block, make sure "logo" is selected when saving, so that it actually updates), and then moved from WP 6.2 to 6.3. That's when the change occured.

### Screenshots, screen recording, code snippet

![image](https://github.com/WordPress/gutenberg/assets/8428432/df610e80-baf2-4c13-b269-6e4158db1b96)
The Site Editor situation is shown on the left, the "regular" block editor on the right. The left icon is way larger while the link's "focus outline" is not.

### Environment info

- WP 6.3-RC2
- Firefox 115.0.3
- Windows 10

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by annezazu** [comment]
Using 6.3 RC3, I'm seeing and replicating the following:

- In the Site Editor, the logo is larger and has a smaller lift in terms of animation for pages and templates.
- In the Post Editor, the logo is smaller and upon hovering, it has more room for animation.

TLDR: The animations and dimensions are different. I don't think this is intentional and I find the post editor animations to be better:

https://github.com/WordPress/gutenberg/assets/26996883/698e3617-5f61-4293-b145-1058e4471591

I'm going to add to the 6.3 board but it seems to be a good candidate for 6.3.x+ as it's not a major bug if it's not able to be quickly addressed.

**Comment by ndiego** [comment]
I feel like this has been the case for a while, and I agree that the post Editor implementation is generally better for the logos I have used. Given that this is not a breaking change and RC3 is today, I think we should punt to 6.3.x

**Comment by annezazu** [comment]
Agreed. Thanks for double checking. Moving to 6.3.x.

**Comment by kevin940726** [comment]
I think this is an intentional design choice made in https://github.com/WordPress/gutenberg/pull/48886. Perhaps @richtabor or @mtias could provide more details about it?

I believe the cut-off comes from the code here:

https://github.com/WordPress/gutenberg/blob/b2e5f0d6556bdd140bee1e728e70f68e11c21ac4/packages/edit-site/src/components/layout/style.scss#L216-L217

...where `$grid-unit-80` is 64px, but the parent container's size is defined here:

https://github.com/WordPress/gutenberg/blob/b2e5f0d6556bdd140bee1e728e70f68e11c21ac4/packages/edit-site/src/components/site-hub/style.scss#L46-L47

which is 60px.

The inconsistency here is probably what makes it "look off".

**Comment by mtias** [comment]
Yeah, the post editor is not opening a sidebar so it already works a bit differently. It'll be reconciled as we reconcile other parts of the site / post editors.

**Comment by ntsekouras** [comment]
The site hub with the site icon this refers to is gone since #79197, the editors show the admin bar instead, so there's no separate site logo rendering to reconcile anymore. Closing, but let's reopen if it's still an issue.
